configure

```diff
@@ -242,6 +242,11 @@ else
   CMAKE_CONFIG_FLAGS="$CMAKE_CONFIG_FLAGS -DTC_BUILD_CAPI=0"
 fi
 
+# If we don't need to build for IOS don't make the flag nothing
+if [[ $with_ios == 0 ]]; then
+  CMAKE_CONFIG_FLAGS="$CMAKE_CONFIG_FLAGS -DTC_BUILD_IOS=0"
```

Comment:
Why do we need this?

On a potentially related note, the existing similar checks in our CMakeLists files tend to be written as  `if(APPLE AND NOT TC_BUILD_IOS)`, no `${}`.

Reply:
Ahh that would work as well, and then we wouldn't explicitly need that